lib/config.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
+/**
```

Comment:
Any comment that starts with `/**` is going to get picked up by our JSDOC API doc parser and i don't think we want this getting picked up. I'd also just say leave this comment block out completely. People can read the name of the file so i don't think we are adding much here.